tests/acl/test_acl.py

```diff
@@ -141,6 +142,9 @@
 PACKETS_COUNT = "packets_count"
 BYTES_COUNT = "bytes_count"
 
+DUT_USER = "admin"
+DUT_PASSWD = "YourPaSsWoRd"
```

Comment:
has this semantics used anywhere else ?

Reply:
I have seen a variation of this in ssh and many other scripts wherein they hardcode the value. But in this case I am making sure that within the setup function, we check what the testbed creds are and updating it accordingly.

Reply:
Changed as requested @abdosi